.travis.yml

```diff
@@ -32,7 +32,9 @@ install:
   - npm install && npm run lint
 
 script:
-  - npm run build_spec && npm run test_mocha && node ./node_modules/markdown-doctest/bin/cmd.js
+  - npm install typescript@2.0.7 && npm run build_spec && npm run test_mocha
+  - npm install typescript@latest && npm run build_spec && npm run test_mocha
```

Comment:
If it's certain new feature like module argumentation, I'd agree that. But for this case, I see no strong reason to not to keep compatibility between 2 versions, especially given custom object is expected to inherit Error Object itself. This test is guarding 'current' change to be safe with previous version of TS, while further version comes out and if decided to introduce version specific breaking changes can be taken out.